src/server/src/main/resources/db/cassandra/016_disable_dclocal_read_repair_chance.cql

```diff
@@ -1,5 +1,5 @@
 --
```

Comment:
Can we change the name of the file?? (eg it would be great to change it to `016_base.cql`, but idk if that's possible)

Reply:
I thought it would be a problem, but it turns out it's not.